src/Panel.vue

```diff
@@ -140,7 +140,7 @@
         return 'panel panel-' + (this.type || (this.inAccordion && this.$parent.type) || 'default')
       },
       headerContent () {
-        return md.renderInline(this.header);
+        return md.render(this.header);
       },
       altContent () {
         return this.alt && md.renderInline(this.alt) || md.renderInline(this.header);
```

Comment:
Change here too. This is used when:
- `minimized` is specified, or
- you click the ✖ on the panel.